src/rofi_rbw/rofi_rbw.py

```diff
@@ -32,6 +37,7 @@ class Action(enum.Enum):
         TYPE_BOTH = 'autotype'
         COPY_USERNAME = 'copy-username'
         COPY_PASSWORD = 'copy-password'
+        COPY_TOTP = 'copy-totp'
 
     def __init__(self) -> None:
         self.args = self.parse_arguments()
```

Comment:
Do you think it makes sense to make the TOTP-copying configurable? Or does one want that every time?

Reply:
I personally use TOTP-copying quite often and since it is one of the last options I don't think it really hurts to have it default on.

Reply:
Having it on by default is one thing (and I think it makes sense), but I'm wondering if there are use cases for being able to turn it off.

Reply:
I am not sure if we are referring to the same thing. Do you want to be able to disable the `Alt-t ` hotkey or to you want to remove the auto-copy function on password (and username) typing?

For the copy-on-type I can image people not wanting their clipboard getting changed, so a toggle might make sense, even though it currently checks if the totp is not empty (which, I think, solves most issues).

Reply:
Ah, sorry. I was talking about copying with `TYPE_PASSWORD` and `TYPE_BOTH`. And I was only wondering if there might be a use case here if someone wants to turn this behavior off.

On the other hand, we can merge it as is and wait if someone opens an issue.

Reply:
In the browser plugin there is AFAIK no option to disable that at the moment, but I think it's fine to not immediately add a new argument. At the other hand it's not too much work to add a argument for it, want me to open a PR?